src/web/resources/report.js

```diff
@@ -395,9 +395,7 @@ const CostAccuracy = new Component('#cost-accuracy', {
         const [initial_pt, best_pt, rest_pts] = benchmark["cost-accuracy"];
         const bits = benchmark["bits"];
         const initial_accuracy = 100*(1 - initial_pt[1]/bits);
-        rest_pts.push(best_pt);
```

Comment:
I'm a little confused here. A quick historic background. Herbie originally didn't return a Pareto curve. It returned one alt, the most accurate one. When we added Pareto mode, it was an optional feature, because it was super slow. For this reason, the "best" alt was stored separately from the others.

Nowadays, Pareto mode is the default, and I don't think we want to expose the concept of "best alt" vs "other alts" to the user. So I think we should could the "best alt" as one of the alternatives, and probably show the table of alternatives even when there's only one.

Reply:
I removed this line because one result (best_pt) was being shown twice.